tests/code_wars/test_song.py

```diff
@@ -0,0 +1,8 @@
+from code_challenges.code_wars.song import Song
+
+
+def test_song():
+    mount_moose = Song("Mount Moose", "The Snazzy Moose")
+
```

Comment:
Should probably test that `mount_moose.title` and `moose.artist` are what you would expect also.

Reply:
Bump